migrations/versions/0439_callback_type_non_nullable.py

```diff
@@ -0,0 +1,23 @@
+"""
+
+Revision ID: 0439_callback_type_non_nullable
+Revises: 0438_request_invite_templates
+Create Date: 2023-12-20 20:58:46.806269
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+revision = "0439_callback_type_non_nullable"
+down_revision = "0438_request_invite_templates"
+
+
+def upgrade():
+    op.alter_column("service_callback_api", "callback_type", existing_type=sa.VARCHAR(), nullable=False)
```

Comment:
Am I right in thinking that this is doing `ALTER TABLE service_callback_api ALTER COLUMN callback_type SET NOT NULL`? Do correct me if wrong though!

If so, this would be covered in https://medium.com/paypal-tech/postgresql-at-scale-database-schema-changes-without-downtime-20d3749ed680#09eb

Which says `adding a not-null constraint to an existing column requires an exclusive lock on the table while a full table scan verifies that no null values exist`. 

Taking a exclusive table lock sounds bad on that table (may not be able to create service callbacks which technically is part of the processing of delivery receipts tasks).

Worth confirming if this is the case and looking into the alternative approach mentioned in the link above for how to do this migration without taking an exclusive table lock.

Reply:
Yes, it does do `ALTER TABLE service_callback_api ALTER COLUMN callback_type SET NOT NULL`. The tables affected are small (433 rows in `service_callback_api` and 1700 in `service_callback_api_history`) but I could just leave the migration out or do it separately in stages if you think it would be better not to do that - it's not really required, it's more of a "nice to have".

Reply:
@idavidmcdonald I've dropped the migration from this PR